Makefile

```diff
@@ -70,7 +70,7 @@ $(SRCPARSER)/flex_lexer.o: $(SRCPARSER)/flex_lexer.cpp $(SRCPARSER)/bison_parser
 	$(CXX) $(LIB_CFLAGS) -c -o $@ $< -Wno-sign-compare -Wno-unneeded-internal-declaration -Wno-register
 
 %.o: %.cpp $(PARSER_CPP) $(LIB_H)
-	$(CXX) $(LIB_CFLAGS) -c -o $@ $<
+	$(CXX) $(LIB_CFLAGS) $(CXXFLAGS) -c -o $@ $<
```

Comment:
We don't use additonal flags in our base repo builds. Unless there's a reason to have them here, I'd ask you to remove them.

Reply:
https://en.wikipedia.org/wiki/CFLAGS

TL;DR: CFLAGS/CXXFLAGS is a Makefile standard, for example for a program

```
#include <iostream>

int main() {
#ifdef FOO
  std::cout << "Foo\n";
#endif
}
```
and, using make's default rule:
```
$ CXXFLAGS=-DFOO make -B cflagtest && ./cflagtest 
g++ -DFOO    cflagtest.cpp   -o cflagtest
Foo
```
It's not that common to have hand-written Makefiles call other hand-written Makefiles - you normally let a build system such as CMake or meson do that for you - but when you do, you need to make to sure you follow this standard and pass along CFLAGS/CXXFLAGS.